application.h

```diff
@@ -91,6 +94,9 @@ class eu07_application {
         is_server() const;
     bool
         is_client() const;
+#ifdef WITH_UART
+    UartStatus uart_status;
```

Comment:
Tak trochę średnio to pasuje. Myślę że lepiej żeby to siedziało w klasie uart_input.

Reply:
Ok.